sklearn/manifold/spectral_embedding_.py

```diff
@@ -288,11 +288,12 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
         laplacian = check_array(laplacian, dtype=np.float64,
                                 accept_sparse=True)
         laplacian = _set_diag(laplacian, 1, norm_laplacian)
+        laplacian = laplacian + 1e-5 * sparse.eye(laplacian.shape[0])
```

Comment:
Okay. Then are you able to directly test these changes?

Reply:
Done; see https://github.com/scikit-learn/scikit-learn/pull/12316#issuecomment-481841935